slices/iptables.yaml

```
@@ -0,0 +1,204 @@
+package: iptables
+
+essential:
+  - iptables_copyright
+
+slices:
+  bins:
+    essential:
+      - iptables_libs
+      - iptables_links
+      - libc6_libs
+      - libip4tc2_libs
+      - libip6tc2_libs
+      - libmnl0_libs
+      - libnetfilter-conntrack3_libs
+      - libnfnetlink0_libs
+      - libnftnl11_libs
+      - libxtables12_libs
+      - netbase_default-hosts
+      - netbase_default-networks
+    contents:
+      /usr/sbin/arptables-nft:
+      /usr/sbin/arptables-nft-restore:
+      /usr/sbin/arptables-nft-save:
+      /usr/sbin/ebtables-nft:
+      /usr/sbin/ebtables-nft-restore:
+      /usr/sbin/ebtables-nft-save:
+      /usr/sbin/ebtables-translate:
+      /usr/sbin/ip6tables-apply:
+      /usr/sbin/ip6tables-legacy:
+      /usr/sbin/ip6tables-legacy-restore:
+      /usr/sbin/ip6tables-legacy-save:
+      /usr/sbin/ip6tables-nft:
+      /usr/sbin/ip6tables-nft-restore:
+      /usr/sbin/ip6tables-nft-save:
+      /usr/sbin/ip6tables-restore-translate:
+      /usr/sbin/ip6tables-translate:
+      /usr/sbin/iptables-apply:
+      /usr/sbin/iptables-legacy:
+      /usr/sbin/iptables-legacy-restore:
+      /usr/sbin/iptables-legacy-save:
+      /usr/sbin/iptables-nft:
+      /usr/sbin/iptables-nft-restore:
+      /usr/sbin/iptables-nft-save:
+      /usr/sbin/iptables-restore-translate:
+      /usr/sbin/iptables-translate:
+      /usr/sbin/nfnl_osf:
+      /usr/sbin/xtables-legacy-multi:
+      /usr/sbin/xtables-monitor:
+      /usr/sbin/xtables-nft-multi:
+
+  # The xlst is used to convert XML configuration into something
+  # iptables can understand, and vice-versa.
+  extras:
```

```suggestion
  converters:
```

or something else a bit more representative